site/content/docs/0.0/utilities/borders.md

```diff
@@ -8,4 +8,315 @@ aliases:
+{{< /example >}}
+
+{{< example class="bd-example-rounded-utils" >}}
+{{< placeholder width="75" height="75" class="rounded-bottom-short" title="Example small rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-start-medium" title="Example default left rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-end-circle" title="Example right completely round image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-start-pill" title="Example left rounded pill image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-tall rounded-top-none" title="Example extra large bottom rounded image" color="#999" >}}
+{{< /example >}}
+
+{{< bootstrap-compatibility >}}
+
+Use the scaling classes for larger or smaller rounded corners. Sizes range from `0` to `5`, and can be configured by modifying the utilities API.
+
+{{< example class="bd-example-rounded-utils" >}}
+{{< placeholder width="75" height="75" class="rounded-0" title="Example non-rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-1" title="Example small rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-2" title="Example default rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-3" title="Example large rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-4" title="Example larger rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-5" title="Example extra large rounded image" color="#999" >}}
+{{< /example >}}
+
+{{< example class="bd-example-rounded-utils" >}}
+{{< placeholder width="75" height="75" class="rounded-bottom-1" title="Example small rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-start-2" title="Example default left rounded image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-end-circle" title="Example right completely round image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-start-pill" title="Example left rounded pill image" color="#999" >}}
+{{< placeholder width="75" height="75" class="rounded-5 rounded-top-0" title="Example extra large bottom rounded image" color="#999" >}}
+{{< /example >}}
+
+{{< /bootstrap-compatibility >}}
+
+<!--## Dividers
+
+{{< example >}}
+<div class="border-top border-light my-3"></div>
+<div class="border-top border-light border-1 my-3"></div>
+<div class="border-top border-dark my-3"></div>
+<div class="border-top border-dark border-1 my-3"></div>
+<div class="border-top my-3"></div>
+<hr class="text-light">
+<hr class="text-body-secondary">
+<hr>
+{{< /example >}}-->
+
+## Drag and drop
+
+Use the `.border-drag` utility to apply the border style for a drop zone.
+
+{{< callout danger >}}
+This utility must not be used in any other context than drag and drop.
```

Comment:
Like in our docs 🙃? 

![Screenshot 2024-09-16 at 08 26 13](https://github.com/user-attachments/assets/8e6c4245-3b61-418c-b525-d6dc8f438069)

Just kidding, we need to check with the designers how to render this section. I liked the fact that the border is a little bit different showing that it's not mandatory to open/read.

Reply:
ahah, do as I say, not as I do :)
You're right, we should ask designer on how to render this kind of content.